modules/os2forms_fbs_handler/os2forms_fbs_handler.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'FBS Handler'
+type: module
+description: 'Provides integration to FBS.'
+package: 'OS2Forms'
+core: 8.x
```

Comment:
this is not needed for drupal 8.8 and newer. Will throw an error in Drupal 10:
https://www.drupal.org/docs/develop/creating-modules/let-drupal-know-about-your-module-with-an-infoyml-file#s-specifying-the-core-version-requirement

Reply:
Fixed